quickstarts/radiant-mlhub-landcovernet.ipynb

```diff
@@ -0,0 +1,574 @@
+      "...\n"
+     ]
+    }
+   ],
+   "source": [
+    "source_imagery_links = item.get_links(rel=LabelRelType.SOURCE)\n",
+    "links_limit = 10\n",
+    "print(f\"Source Imagery Links: {len(source_imagery_links)}\")\n",
+    "for link in it.islice(source_imagery_links, links_limit):\n",
+    "    print(f\"- {link.href}\")\n",
+    "if len(source_imagery_links) > links_limit:\n",
+    "    print(\"...\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can see that there are 72 different source images that can be associated with these labels. Let's grab the STAC Item for the first one so we can download the RGB band assets for that image. Because the Radiant MLHub API requires authentication to retrieve Items, we cannot use the standard PySTAC methods for resolving STAC Objects. Instead, we will use the custom `requests.Session` instance we created above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Item ID: ref_landcovernet_v1_source_38PKT_29_20180101\n",
+      "Assets:\n",
+      "- Asset Key: B01\n",
+      "  Bands:Coastal Aerosol\n",
+      "- Asset Key: B02\n",
+      "  Bands:Blue\n",
+      "- Asset Key: B03\n",
+      "  Bands:Green\n",
+      "- Asset Key: B04\n",
+      "  Bands:Red\n",
+      "- Asset Key: B05\n",
+      "  Bands:Vegetation Red Edge\n",
+      "- Asset Key: B06\n",
+      "  Bands:Vegetation Red Edge\n",
+      "- Asset Key: B07\n",
+      "  Bands:Vegetation Red Edge\n",
+      "- Asset Key: B08\n",
+      "  Bands:NIR\n",
+      "- Asset Key: B09\n",
+      "  Bands:Water Vapour\n",
+      "- Asset Key: B11\n",
+      "  Bands:SWIR\n",
+      "- Asset Key: B12\n",
+      "  Bands:SWIR\n",
+      "- Asset Key: B8A\n",
+      "  Bands:Narrow NIR\n",
+      "- Asset Key: CLD\n",
+      "  Bands:Cloud Mask\n",
+      "- Asset Key: SCL\n",
+      "  Bands:Scene Classification Layer\n"
+     ]
+    }
+   ],
+   "source": [
+    "image_link = source_imagery_links[0]\n",
+    "\n",
+    "response = session.get(image_link.href)\n",
+    "\n",
+    "image_item = Item.from_dict(response.json())\n",
+    "print(f\"Item ID: {image_item.id}\")\n",
+    "print(\"Assets:\")\n",
+    "for asset_key, asset in image_item.assets.items():\n",
+    "    print(f\"- Asset Key: {asset_key}\")\n",
+    "    asset_eo_ext = EOExtension.ext(asset)\n",
+    "    if asset_eo_ext.bands is not None:\n",
+    "        band_names = \", \".join(band.common_name for band in asset_eo_ext.bands)\n",
+    "        print(f\"  Bands:{band_names}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since we are interested in the RGB bands for this image, we will download the `\"B04\"`, `\"B03\"`, and `\"B02\"` assets."
```

Comment:
Our primary use-case up to this point has been allowing users to download data archives so they can work on their own systems, but that could certainly change with a tighter integration to the Planetary Computer. I would be fine with updating this tutorial to end with a demo of using `rioxarray` to read the data assets.

Reply:
Either is fine by me. Given the fact that the next section also downloads data for the archive example, I think this is probably fine. But it might be worth revisiting this as we figure out exactly doing ML on the Planetary Computer looks like.